app/models/riot.rb

```diff
@@ -1,7 +1,7 @@
 class Riot
   include HTTParty
   debug_output $stdout
-  default_params :api_key => '82ced787-5473-4e7f-938f-c35a1f628ade'
+  default_params :api_key => ENV['RIOT_API_KEY']
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.